src/components/core/PivotTable.tsx

```diff
@@ -901,8 +906,12 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
             { attributes: [], all: [] }
         );
 
-        return sortedColumnIndexes.attributes[0] === 0 && sortedColumnIndexes.all.length === 1 ||
-            sortedColumnIndexes.all.length === 0;
+        const i = sortedColumnIndexes.attributes[0] === 0
```

Comment:
Remove variable `i`
And i think the condition is bit messy and i don't understand it. I would split them to some logical blocks like
```
isFirstConditionTrue = sortedColumnIndexes.attributes[0] === 0;
isSecondConditionTrue = ...;
return isFirstConditionTrue || isSecondConditionTrue
```